docs/reference/semaphore-changelog.md

```diff
@@ -8,6 +8,11 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 ### Week of August 17, 2020
+**(New) Install-package command**  
+"Toolbox" command line tools were extended to include `install-package` utility.
+The `install-package` command help with package installations by automatically 
```

Comment:
The `install-package` command helps / can help (not "The `install-package` command help")